lib/crawlfs.js

```diff
@@ -1,21 +1,37 @@
 'use strict'
-const fs = process.versions.electron ? require('original-fs') : require('fs')
-const glob = require('glob')
 
-module.exports = function (dir, options, callback) {
-  const metadata = {}
-  return glob(dir, options, function (error, filenames) {
-    if (error) { return callback(error) }
-    for (const filename of filenames) {
-      const stat = fs.lstatSync(filename)
+const pify = require('pify')
+
+const fs = pify(process.versions.electron ? require('original-fs') : require('fs'))
```

Comment:
I'd much rather use `fs-extra` but I don't think you can override the `fs` module that's used, from what I could tell. I'd be happy if someone could prove me wrong, because I'd get rid of `rimraf`, `mkdirp`, and a homebrewed `copyFile` (which we can probably get rid of anyway, when we move to Node 8).